README.md

```diff
@@ -1 +1,21 @@
 # Requests
+
+# Draft of the Rules
+
+- Anyone can join @morganey-lang organization as a Member
+
+- If someone have an Open Sourced Morganey related project, they can join the organization as a Project Leader by transfering the ownership of the project to @morganey-lang. What that means:
+
+    - The Project Leader preserves all of the legal rights on the transfered project. From the technical point of view I will make sure that after the transfering the ownership the Project Leader has enough control over their repo.
```

Comment:
> Usually we don't care about the legal stuff (because it sucks); personally I just define the copyright holder as some non-anthropomorphous entity like "Morganey contributors" that will automatically assimilate every contributor, so no problems will arise for a while.

Yeah, I completely agree with you. I did actually want to talk about either. We are engineers. Not lawyers. The whole idea is that @morganey-lang provides a... how'd you call that?.. community hosting? We just host people's repos and help them to collaborate. That's it. That's why we transfer repos back if asked. Anything else is outside of the @morganey-lang scope. So yeah, I'm gonna remove the legal stuff.

> Although the one requirement I'd really like to add is a license requirement.

I was thinking about that too. Simply forgot to put that up. :) Thanks for reminding me!